README.md

```diff
@@ -223,6 +234,10 @@ same major line. Should you need to upgrade to a new major, use an explicit
   not to lookup on the remote registry for the latest version of the selected
   package manager.
 
+- `COREPACK_ENABLE_URL_VERSION_FOR_KNOWN_PM` can be set to `1` to allow use of
+  custom URLs to load a package manager known by Corepack (`yarn`, `npm`, and
+  `pnpm`).
+
```

Comment:
Either we should enable this by default or, if it's for a security reason, add `unsafe` somewhere in the key (and document the drawback):

```
COREPACK_ENABLE_UNSAFE_CUSTOM_URLS=1
```